src/components/panel/panel.js

```diff
@@ -355,6 +384,24 @@ angular
  * @param {!MdPanelPosition} position
  */
 
+/**
+ * @ngdoc method
+ * @name MdPanelRef#addToGroup
+ * @description
+ * Adds a panel to a group. The group must be created ahead of time.
```

Comment:
@ErinCoughlan It is not a problem moving to supporting a panel being in multiple groups. Should I build this in?